koku/reporting/provider/ocp/models.py

```diff
@@ -340,7 +340,7 @@ class Meta:
 
     supplementary_project_monthly_cost = JSONField(null=True)
 
-    monthly_cost_type = models.TextField(null=True, choices=MONTHLY_COST_TYPES)
+    cost_type = models.TextField(null=True, choices=COST_TYPES)
```

Comment:
Yeah, I've been debating that. I know this violates our migration rule. I'd be happy just to overload the monthly_cost_type term to avoid a migration. If we do the standard way we'll also need a data migration to populate the new column from the old.

Reply:
🔧